### ISSUE TITLE
Add link to re-run report against LightHouse when running "locally" locally, in PR comment

### ISSUE THREAD

**Issue by Vadorequest** [foo-software/lighthouse-check-action#34]
![image](https://user-images.githubusercontent.com/3807458/88486286-f9ad4d00-cf7c-11ea-8bb7-e9d746d377f7.png)

I could use a link close by the url that's been tested with "Run again" text, so that I can run the test again real quick to get a full report.

It could link to either 

- https://web.dev/measure/
- https://lighthouse-metrics.com/

**Comment by adamhenson** [comment]
This is a great idea, but instead we could link to our page at https://www.foo.software/lighthouse. We could add query params for `url` and `device` to trigger it automatically as soon as that page loads.

**Comment by Vadorequest** [comment]
Great! This way we could even have shortcuts for each device (one link per device).
I assume it's not "too much", and there are only 2 devices available?

**Comment by adamhenson** [comment]
@Vadorequest - I made an update released in [v2.0.5](https://github.com/foo-software/lighthouse-check-action/releases/tag/v2.0.5). I don't think we'll be able to support a link that automatically triggers an audit, but we added a "desktop" and "mobile" radio button on the home page. See the comment example in #31 

<img width="564" alt="Screen Shot 2020-07-27 at 12 22 13 AM" src="https://user-images.githubusercontent.com/747743/88503339-6cc7bb00-cf9f-11ea-9fa2-4e83f395c5a7.png">

<img width="332" alt="Screen Shot 2020-07-27 at 12 24 07 AM" src="https://user-images.githubusercontent.com/747743/88503376-81a44e80-cf9f-11ea-803d-cc9ea3a893ce.png">

**Comment by Vadorequest** [comment]
I believe you meant https://github.com/foo-software/lighthouse-check-action/pull/35 instead of #35 

Gonna check this out, thanks for releasing that fast!

**Comment by Vadorequest** [comment]
@adamhenson https://github.com/UnlyEd/next-right-now/pull/141#commitcomment-40927854 works like a charm

Maybe it'd be interesting to add the ability to customise the Github comment, but that'd be a different need. (point being to add our own urls to the UI)

Or, completely change the layout and rethink the way to present information, to avoid taking so much space and optimise it a bit.

![image](https://user-images.githubusercontent.com/3807458/88553691-d68aa800-d025-11ea-9afd-42d725df06c1.png)

I could see a layout where the urls don't take that much space (optimise links when they all share the same base url to highlight the difference instead of using the entire url _(e.g: use `[/fr](https://xxx/fr)` instead of `[https://xxx/fr](https://xxx/fr)`)_.

Then, the scores could be part of the table too, as third column.

That would look like this:
![image](https://user-images.githubusercontent.com/3807458/88554226-8d872380-d026-11ea-905b-2f37dbec66cc.png)

The full base url could be specified above/below only once and save lots of space.

Maybe this alternative layout could become a new option, as it makes a lot of sense when there are many reports to display. (avoid breaking change UI, if that's important)

**Comment by adamhenson** [comment]
Thanks @Vadorequest. I agree - we can re-think the comment layout. I’m not sure I’d want to support custom comments at this time. Omitting the base URL is an interesting idea. There could be other options too. I’ll open a separate issue for that.

**Comment by adamhenson** [comment]
Created an issue in `lighthouse-check` since that's where the changes would be made - regarding comment layout. foo-software/lighthouse-check/issues/40
